cmd/soci/commands/image/rpull.go

```diff
@@ -81,15 +81,7 @@ command.
 			return fmt.Errorf("please provide an image reference")
 		}
 
-		sociIndexDigest := context.String("soci-index-digest")
```

Comment:
why delete?

Reply:
Since we are not checking `sociIndexDigest == ""` anymore, avoided creating a new variable for it.